gdcdictionary/schemas/gdc_release.yaml

```diff
@@ -0,0 +1,77 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "gdc_release"
+title: GDC Release 
+type: object
+namespace: http://gdc.nci.nih.gov
+category: administrative 
+program: 'GDC'
+project: 'MISC'
+description: >
+  Internal node to store different GDC releases.
+additionalProperties: false
+submittable: false
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+
+links:
+  - name: cases
+    backref: gdc_releases
+    label: describes
+    target_type: case
+    multiplicity: many_to_one
```

Comment:
Wouldn't many cases be in a single release? Also how can a case be in multiple releases?

Reply:
Scratch that. There was decision made on Friday to make a new `Root` node to which this node will be directly connected as well as all `Program` nodes. Check with @christiandompierre @philloooo for more info

Reply:
* the part about connecting `Program` is being debated